### PR TITLE
Unhardcode LightPaletteRotator properties

### PR DESCRIPTION
**Pull request by reaperrr**
Plus some style fixes.
Best testcase for potential regressions: RA Flame Turret.

Also, moved legacy map import commands to UtilityCommands subfolder.

### CODE REVIEW

**Review by obrakmann — APPROVED** [review]
👍

File: OpenRA.Mods.Cnc/Traits/PaletteEffects/LightPaletteRotator.cs [hunk 1]
```diff
@@ -18,8 +18,24 @@ namespace OpenRA.Mods.Cnc.Traits
...
 		public readonly HashSet<string> ExcludePalettes = new HashSet<string>();
 
+		[Desc("'Speed' at which the effect cycles through palette indices.")]
+		public readonly float TimeStep = .5f;
+
+		[Desc("Palette index to map to rotating colors.")]
+		public readonly int ModifyIndex = 103;
+
+		[Desc("First palette index of rotating color range.")]
+		public readonly int RotationStartIndex = 230;
```
Comment (pchote) [inline OpenRA.Mods.Cnc/Traits/PaletteEffects/LightPaletteRotator.cs]: Would there be any value in specifying this as a list of indices to cycle through instead a contiguous range?

Comment (GraionDilach) [inline OpenRA.Mods.Cnc/Traits/PaletteEffects/LightPaletteRotator.cs]: Can I say yes? I'd like to say very much so.

**Comment by reaperrr** [comment]
Updated. Was easier than expected and actually simplifies the code (no reverse-rotate magic needed).

**Review by atlimit8 — APPROVED** [review]
The new version should get two votes to merge according to policy.

**Review by obrakmann — APPROVED** [review]
👍 still stands.

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/1c1a3ab1e563c24e05c890ecaf26b20bd285e492)

**Pull request merged**